subt_ign/launch/competition.ign

```diff
@@ -150,104 +285,191 @@
             entity_type="world"
             filename="libGameLogicPlugin.so"
             name="subt::GameLogicPlugin">
-    <!-- The collection of artifacts to locate -->
+      <!-- The collection of artifacts to locate -->
+      <world_name><%= $worldName %></world_name>
 
-    <world_name><%= $worldName %></world_name>
+      <duration_seconds><%= $durationSec %></duration_seconds>
 
-    <duration_seconds><%= $durationSec %></duration_seconds>
+      <logging>
+        <!-- Use the <path> element to control where to record the log file.
+             The HOME path is used by default -->
+        <path>/tmp/ign/logs</path>
+        <filename_prefix>subt_cave</filename_prefix>
+      </logging>
 
-    <logging>
-      <!-- Use the <path> element to control where to record the log file.
-           The HOME path is used by default-->
-      <path>/tmp/ign/logs</path>
-      <filename_prefix>subt_tunnel</filename_prefix>
-    </logging>
-
-    <artifact>
-      <name>backpack_1</name>
-      <type>TYPE_BACKPACK</type>
-    </artifact>
-    <artifact>
-      <name>backpack_2</name>
-      <type>TYPE_BACKPACK</type>
-    </artifact>
-    <artifact>
-      <name>backpack_3</name>
-      <type>TYPE_BACKPACK</type>
-    </artifact>
-    <artifact>
-      <name>backpack_4</name>
-      <type>TYPE_BACKPACK</type>
-    </artifact>
-
-    <artifact>
-      <name>drill_1</name>
-      <type>TYPE_DRILL</type>
-    </artifact>
-    <artifact>
-      <name>drill_2</name>
-      <type>TYPE_DRILL</type>
-    </artifact>
-    <artifact>
-      <name>drill_3</name>
-      <type>TYPE_DRILL</type>
-    </artifact>
-    <artifact>
-      <name>drill_4</name>
-      <type>TYPE_DRILL</type>
-    </artifact>
-
-    <artifact>
-      <name>extinguisher_1</name>
-      <type>TYPE_EXTINGUISHER</type>
-    </artifact>
-    <artifact>
-      <name>extinguisher_2</name>
-      <type>TYPE_EXTINGUISHER</type>
-    </artifact>
-    <artifact>
-      <name>extinguisher_3</name>
-      <type>TYPE_EXTINGUISHER</type>
-    </artifact>
-    <artifact>
-      <name>extinguisher_4</name>
-      <type>TYPE_EXTINGUISHER</type>
-    </artifact>
-
-    <artifact>
-      <name>phone_1</name>
-      <type>TYPE_PHONE</type>
-    </artifact>
-    <artifact>
-      <name>phone_2</name>
-      <type>TYPE_PHONE</type>
-    </artifact>
-    <artifact>
-      <name>phone_3</name>
-      <type>TYPE_PHONE</type>
-    </artifact>
-    <artifact>
-      <name>phone_4</name>
-      <type>TYPE_PHONE</type>
-    </artifact>
-
-    <artifact>
-      <name>rescue_randy_1</name>
-      <type>TYPE_RESCUE_RANDY</type>
-    </artifact>
-    <artifact>
-      <name>rescue_randy_2</name>
-      <type>TYPE_RESCUE_RANDY</type>
-    </artifact>
-    <artifact>
-      <name>rescue_randy_3</name>
-      <type>TYPE_RESCUE_RANDY</type>
-    </artifact>
-    <artifact>
-      <name>rescue_randy_4</name>
-      <type>TYPE_RESCUE_RANDY</type>
-    </artifact>
-  </plugin>
+      <artifact>
```

Comment:
I am no up to date on which circuit/world has which artifacts. With these changes, it looks like they would all have the same artifacts. Is that okay?

Reply:
It's okay to list them here. The artifact list in the launch files capture all the possible artifacts. The world SDF files contain the artifacts for each world.